src/Root.tsx

```diff
@@ -0,0 +1,22 @@
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+import { HomePage } from './components/HomePage';
+
+export const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="/home" element={<Navigate to="/" replace />} />
+
+        <Route index element={<HomePage />} />
+        <Route path="/people">
+          <Route index element={<PeoplePage />} />
+          <Route path=":slug" element={<PeoplePage />} />
```

Comment:
The route ':slug' under '/people' uses the same component 'PeoplePage'. If this is intentional for handling dynamic routes, ensure that 'PeoplePage' can handle different slugs appropriately.